torch/_refs/__init__.py

```diff
@@ -3525,7 +3525,7 @@ def transpose(a: TensorLikeType, dim0: int, dim1: int) -> TensorLikeType:
 swap_axes = transpose
 
 
-@register_decomposition(torch.ops.aten.unfold)
+@register_decomposition(torch.ops.aten.unfold, disable_meta=True)
```

Comment:
can we say what's wrong about this decomposition?

Reply:
being fixed in https://github.com/pytorch/pytorch/pull/86428